### PR TITLE
Documentation rework for 3.1 release

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
test this please

File: docs/src/docs/asciidoc/user/107.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 ---
 = The Ehcache 3.x JSR-107 Provider
-ifndef::sourcedir[]
+ifndef::sourcedir31[]
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/107.adoc]: If we're going C style on the includes shouldn't this be `ifndef::common_adoc[]`?

File: docs/src/docs/asciidoc/user/107.adoc [hunk 2]
```diff
@@ -9,10 +9,10 @@ ifdef::notBuildingForSite[]
...
 endif::notBuildingForSite[]
 
-== Getting started with Ehcache 3.0 & JSR-107
+== Getting started with Ehcache 3 & JSR-107
 
-There is nothing special for you to do to use Ehcache 3.0 as the caching provider for your application: add the
-`ehcache-3.0.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
+There is nothing special for you to do to use Ehcache 3 as the caching provider for your application: add the
+`ehcache-3.1.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
 go:
```
Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/107.adoc]: should we think about putting `ehcache-3.1.0.jar` on it's own line - futureproofing?

**Comment by chrisdennis** [comment]
I assume you didn't mean to update the readme files as part of this?

**Comment by jhouserizer** [comment]
Generally looks good to me (noting Chris' comments).

**Comment by jhouserizer** [comment]
Are we going to try to get this merged before doing other doc (content) work?  Or should we proceed with the latter and deal with re-basing?

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
Planning on making the requested changes soon ... probably good to wait instead of having crazy rebase

File: docs/src/docs/asciidoc/user/107.adoc (hunk 1 above)
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/107.adoc]: Sure, will improve on master after the release

**Comment by ljacomet** [comment]
Updating the readme will happen at / just after release time

**Comment by ljacomet** [comment]
Failure is unrelated - this should be good to merge

**Pull request merged**